src/status_im/subs/wallet/activities.cljs

```diff
@@ -69,9 +112,11 @@
  :<- [:wallet/network-details]
  (fn [[activities current-viewing-account-address network-details]]
    (let [chain-id->network-name (update-vals (group-by :chain-id network-details)
-                                             (comp :network-name first))]
-     (->> current-viewing-account-address
-          (get activities)
+                                             (comp :network-name first))
+         address-activities     (->> (get activities current-viewing-account-address)
+                                     (vals)
+                                     (sort :timestamp))]
+     (->> address-activities
```

Comment:
Since the activities now are indexed by hash, this sub had to be updated to properly extract the data